packages/cli/test/integration/shared/test-db.ts

```diff
@@ -56,71 +50,13 @@ export async function terminate() {
 	Db.connectionState.connected = false;
 }
 
-// Can't use `Object.keys(entities)` here because some entities have a `Entity` suffix, while the repositories don't
-const repositories = [
-	'AnnotationTag',
-	'AuthIdentity',
-	'AuthProviderSyncHistory',
-	'Credentials',
-	'EventDestinations',
-	'Execution',
-	'ExecutionAnnotation',
-	'ExecutionData',
-	'ExecutionMetadata',
-	'InstalledNodes',
-	'InstalledPackages',
-	'Project',
-	'ProjectRelation',
-	'Role',
-	'ProcessedData',
-	'Project',
-	'ProjectRelation',
-	'Settings',
-	'SharedCredentials',
-	'SharedWorkflow',
-	'Tag',
-	'TestDefinition',
-	'TestMetric',
-	'TestRun',
-	'User',
-	'Variables',
-	'Webhook',
-	'Workflow',
-	'WorkflowHistory',
-	'WorkflowStatistics',
-	'WorkflowTagMapping',
-	'ApiKey',
-	'Folder',
-	'InsightsRaw',
-	'InsightsMetadata',
-	'InsightsByPeriod',
-] as const;
+type EntityName = keyof typeof entities | 'InsightsRaw' | 'InsightsByPeriod' | 'InsightsMetadata';
 
-/**
- * Truncate specific DB tables in a test DB.
- */
-export async function truncate(names: Array<(typeof repositories)[number]>) {
-	for (const name of names) {
-		let RepositoryClass: Class<Repository<object>>;
+export async function truncate(names: EntityName[]) {
+	const connection = Container.get(Connection);
 
-		const fileName = `${kebabCase(name)}.repository`;
-		const paths = [
-			`@/databases/repositories/${fileName}.ee`,
-			`@/databases/repositories/${fileName}`,
-		];
-
-		for (const extension of loadedExtensions) {
-			paths.push(
-				`@/modules/${extension}/database/repositories/${fileName}`,
-				`@/modules/${extension}/database/repositories/${fileName}.ee`,
-			);
-		}
-
-		RepositoryClass = (await Promise.any(paths.map(async (path) => await import(path))))[
-			`${name}Repository`
-		];
-
-		await Container.get(RepositoryClass).delete({});
+	for (const name of names) {
+		await connection.getRepository(name).delete({});
```

Comment:
if we are really switching over to using entities here, we might as well change `trucate` to accept the actual entity classes. 
That way we also don't need to maintain the `EntityName` type.

Reply:
That will require touching every argument on every call to `truncate`. I'd prefer to do this separately as this PR is big enough already?